packages/core/src/Stores/notification-store.js

```diff
@@ -327,7 +327,7 @@ export default class NotificationStore extends BaseStore {
                 if (is_p2p_visible) {
                     this.addNotificationMessage(this.client_notifications.dp2p);
                 } else {
-                    this.removeNotificationMessageByKey({ key: this.client_notifications.dp2p.key });
+                    this.removeNotificationMessageByKey({ key: this.client_notifications?.dp2p?.key });
```

Comment:
we shouldn't be calling `removeNotificationMessageByKey` if the key is undefined